docs/configuration.md

```diff
@@ -1664,6 +1664,14 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.<particular protocol>.port</code></td>
```

Comment:
Yes that's what I'm referring to. I would still use `spark.ssl.port` as the key for this section for consistency. It's OK to maybe add some text reminding the reader that this port can and probably should be set separately for each service.